core/app/[locale]/(default)/account/[tab]/_components/product-snippet.tsx

```diff
@@ -0,0 +1,149 @@
+import { useFormatter } from 'next-intl';
+
+import { graphql, ResultOf } from '~/client/graphql';
+import { BcImage } from '~/components/bc-image';
+import { PricingFragment } from '~/components/product-card';
+import { cn } from '~/lib/utils';
+import { Link } from '~/navigation';
+
+export const ProductCardFragment = graphql(
+  `
+    fragment ProductCardFragment on Product {
+      entityId
+      name
+      defaultImage {
+        altText
+        url: urlTemplate
+      }
+      path
+      brand {
+        name
+      }
+
+      ...PricingFragment
+    }
+  `,
+  [PricingFragment],
+);
+
+interface Props {
+  product: ResultOf<typeof ProductCardFragment>;
+  imageSize?: 'tall' | 'wide' | 'square';
+  brandSize?: string;
+  productSize?: string;
+  imagePriority?: boolean;
+}
+
+export const ProductSnippet = ({
```

Comment:
Can you reuse `ProductCard` instead?

Reply:
well, it was my thought but we use kind of a simpler version for Orders pages. Besides that in Orders we have  a different design as well. We've discussed with Turner and Product title is smaller than on PLP since it is more for management things rather marketing as on PLP. I also try re-use ProductSnippet on Order Details Page which has extended version ( it has little bit different layout and product options). Taking this into account I've thought that creating a specific version of ProductCard for account pages makes more sense than to extend and add props for our ProductCard.